es-app/src/SystemData.cpp

```diff
@@ -186,7 +272,7 @@ bool SystemData::loadConfig()
 	pugi::xml_document doc;
 	pugi::xml_parse_result res = doc.load_file(path.c_str());
 
-	if(!res)
+	if (!res)
 	{
```

Comment:
I noticed this and other whitespace changes. Although the formatting of es isn't ideal, this shouldn't be done as part of this PR. Please can you include code changes only.